src/types.ts

```diff
@@ -0,0 +1,9 @@
+import type { HashName } from 'multihashes'
+
+export interface Digest {
+  (data: Uint8Array): Promise<Uint8Array> | Uint8Array
+}
+
+// This works around the TS bug which emits bad import paths
```

Comment:
Is this needed after the https://github.com/multiformats/js-multihash/pull/113 followup ? Should we block on that?